dataprofiler/data_readers/data_utils.py

```diff
@@ -399,9 +401,9 @@ def _decode_is_valid(encoding):
                                         chunk_size=512, threshold=0.2,
                                         cp_isolation=None, cp_exclusion=None,
                                         preemptive_behaviour=True, explain=False)
-                result = result.best().first()
+                result = result.best()
             if result:
-                encoding = result.encoding
+                encoding = result.first().encoding
```

Comment:
What if there's not first?

Reply:
Currently, this is error'ing bc result itself is returning `null`, hence result.first() doesn't exist. maybe ill check for both bc i might be on a. newer version